es6/server/auth/check-auth.js

```diff
@@ -0,0 +1,15 @@
+import jwt from 'jsonwebtoken';
+
+export default (req, res, next) => {
+    try{
+        const token = req.headers.authorization;
+        const decoded = jwt.verify(token, process.env.JWT_KEY);
+        req.userData = decoded;
+        next();
+    }
+    catch(error){
+        return res.status(401).json({
+            message: 'Authentication Failed!'
```

Comment:
I think this message can be more explanatory. `invalid token.`?
what happens when the user has no token?

Reply:
noted, thanks